packages/mds-db/index.ts

```diff
@@ -1163,14 +1152,14 @@ async function readTrips(params: ReadEventsQueryParams & { skip: DBVal; take: DB
     if (!isTimestamp(min_end_time)) {
       throw new Error(`invalid min_end_time ${min_end_time}`)
     } else {
-      conditions.push(`min_end_time >= ${vals.add(min_end_time)}`) // FIXME confirm >=
+      conditions.push(`min_end_time >= ${vals.add(min_end_time)}`) 
```

Comment:
I'm pretty sure `min_end_time` and `max_end_time` are not actual columns. This is probably a bug waiting to happen. Likely this should be `timestamp >= ${vals.add(min_end_time)}` ... this isn't the only instance of this.

Reply:
Yeah, you're right.

Reply:
I just went through the code and that was the only instance I could find of `min_end_time` and `max_end_time` being used instead of `timestamp`, actually.